monai/transforms/croppad/array.py

```diff
@@ -272,9 +278,13 @@ def __call__(self, img: Union[np.ndarray, torch.Tensor]):
 class CenterSpatialCrop(Transform):
     """
     Crop at the center of image with specified ROI size.
+    If a dimension of the expected ROI size is bigger than the input image size, will not crop that dimension.
+    So the cropped result may be smaller than expected ROI, and the cropped results of several images may not
+    have exactly same shape.
 
     Args:
         roi_size: the spatial size of the crop region e.g. [224,224,128]
+            if a dimension of ROI size is bigger than image size, will not crop that dimension of the image.
```

Comment:
Hi @Nic-Ma , do you think that we can add an example here?
e.g. 
```
example = torch.randn([3, 3, 3, 3])
cropper = CenterSpatialCrop(roi_size=[4, 2, 2])
print(cropper(example).shape)
#  torch.Size([3, 3, 2, 2])
```

Reply:
added example in the latest commit.

Thanks.